src/test/java/io/jenkins/plugins/appcenter/task/internal/DistributeResourceTaskTest.java

```diff
@@ -81,4 +81,4 @@ public void should_ReturnException_When_RequestIsUnSuccessful() {
         assertThat(exception).hasCauseThat().hasCauseThat().isInstanceOf(HttpException.class);
         assertThat(exception).hasCauseThat().hasCauseThat().hasMessageThat().isEqualTo("HTTP 500 Server Error");
     }
-}
\ No newline at end of file
+}
```

Comment:
As well as these tests there is probably also a specific test that you can write to assert the correct notify testers flag is sent to the server depending on what the user has selected.

Reply:
For sure. I'll try and add that

Reply:
I've added these now.